modules/pantheon/pantheon_apachesolr/pantheon_apachesolr.module

```diff
@@ -157,6 +157,17 @@ function pantheon_apachesolr_curlopts() {
   );
 }
 
+function pantheon_apachesolr_certs_dir() {
+  if (isset($_ENV['PANTHEON_ENVIRONMENT'])) {
+    return $_SERVER['HOME'] . '/certs';
+  }
+  return '../certs';
```

Comment:
Pantheon only please. This whole module should deactivate itself if `$_ENV['PANTHEON_ENVIRONMENT']` is undefined.